dev-docs/bidders/e_volution.md

```diff
@@ -12,6 +12,7 @@ pbs_app_supported: true
 usp_supported: true
 schain_supported: true
 pbjs_version_notes: in 6.8+
+userIds: id5Id
```

Comment:
There's also a new flag for your bidder documentation introduced in #3694

`multiformat_supported` . Possible values are

* `will-bid-on-any` - bidder will bid on any media type
* `will-bid-on-one` - bidder will prioritize one media type and ignores the others
* `will-not-bid` - bidder will not bid at all

This helps publishers / sales houses to know up front how adapter will behave in a multi-format environment